src/util/check.js

```diff
@@ -24,8 +24,13 @@ function getContributorsPage(url) {
       },
     })
     .then(res => {
-      const body = JSON.parse(res.body)
-      const contributorsIds = body.map(contributor => contributor.login)
+      let body, contributorsIds
+      try {
+        body = JSON.parse(res.body)
+        contributorsIds = body.map(contributor => contributor.login)
+      } catch (error) {
+        throw new Error(error)
```

Comment:
Why do we catch if we're going to throw anyways? 🤔

Reply:
oh ok should i use chalk to display the error